python/servo/devenv_commands.py

```diff
@@ -270,3 +271,18 @@ def ndk_gdb(self, release, target):
             "--verbose",
         ], env=env)
         return p.wait()
+
+    @Command('merge-bencher-reports', description='Merges BMF JSONs', category='devenv')
+    @CommandArgument('inputs', default=[], nargs="...", help="BMF JSON files to merge")
+    @CommandArgument('--bmf-output', help="Specify BMF JSON output file")
+    def merge_bencher_reports(self, inputs: list[str], bmf_output: str):
+        output: dict[str, object] = dict()
+        for input_file in inputs:
+            with open(input_file, 'r', encoding='utf-8') as f:
+                output = json.load(f) | output
+
+        if bmf_output:
+            with open(bmf_output, 'w', encoding='utf-8') as f:
+                json.dump(output, f, indent=4)
+        else:
+            print(json.dumps(output, indent=4))
```

Comment:
Where is this merge command used? It feels almost a bit too specific for a particular task and also this is a few lines of Python. It's hard to know if this is appropriate for `mach` without having the full context.

Reply:
TBH I was also thinking the same thing, maybe this and test-size should rather go to etc/ci a part of https://github.com/servo/servo/pull/33261

Reply:
Yeah, that seems reasonable! That way the scripts will be available if / when we stop using `mach`.